storey/flow.py

```diff
@@ -640,26 +657,63 @@ def _v3io_parse_get_item_response(response_body):
     for name, type_to_value in response_object.items():
         val = None
         for typ, value in type_to_value.items():
-            if typ == 'S' or typ == 'BOOL':
-                val = value
-            elif typ == 'N':
-                if _non_int_char_pattern.search(value):
-                    val = float(value)
-                else:
-                    val = int(value)
-            elif typ == 'B':
-                val = base64.b64decode(value)
-            elif typ == 'TS':
-                splits = value.split(':', 1)
-                secs = int(splits[0])
-                nanosecs = int(splits[1])
-                val = datetime.utcfromtimestamp(secs + nanosecs / 1000000000)
-            else:
-                raise V3ioError(f'Type {typ} in get item response is not supported')
+            val = _convert_nginx_to_python_type(typ, value)
         response_object[name] = val
     return response_object
 
 
+def _convert_nginx_to_python_type(typ, value):
+    if typ == 'S' or typ == 'BOOL':
+        return value
+    elif typ == 'N':
+        if _non_int_char_pattern.search(value):
+            return float(value)
+        else:
+            return int(value)
+    elif typ == 'B':
+        return base64.b64decode(value)
+    elif typ == 'TS':
+        splits = value.split(':', 1)
+        secs = int(splits[0])
+        nanosecs = int(splits[1])
+        return datetime.utcfromtimestamp(secs + nanosecs / 1000000000)
```

Comment:
`nanosecs / 1000000000` is always zero, isn't it?

Reply:
That's just your code that i moved 😅

Reply:
Oh, actually in Python is converts to float automatically, so I guess it's ok.